File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c++/jcarlosbm.cpp
//
//  main.cpp
//  FizzBuzz
//
//  Created by Juan Carlos Bonilla on 12/15/23.
//

#include <iostream>

int main() {

    for(short i = 1; i <= 100; i++){
        if (i%3 == 0 && i%5 == 0) std::cout << "fizzbuzz" << std::endl;
        else if(i%3 == 0) std::cout << "fizz" << std::endl;
        else if(i%5 == 0) std::cout << "buzz" << std::endl;
        else std::cout << i << std::endl;
    }
    
    return 0;
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/swift/jcarlosbm.swift
//
//  main.swift
//  FizzBuzz
//
//  Created by Juan Carlos Bonilla on 12/15/23.
//

import Foundation

for i in 1...100{
    if i%3 == 0 && i%5 == 0 {print("fizzbuzz")}
    else if i%3 == 0 {print("fizz")}
    else if i%5 == 0 {print("buzz")}
    else {print(i)}
}

